webapp-django/crashstats/crashstats/tests/test_views.py

```diff
@@ -446,7 +434,6 @@ def mocked_processed_crash_get(**params):
         ) in content
         # Should not be able to see sensitive key from stackwalker JSON
         assert "&#34;sensitive&#34;" not in content
```

Comment:
"sensitive" isn't in the stackwalker output anymore so this will always be true.